### DISCUSSION TITLE
How can I contribute?

### DISCUSSION THREAD

**Discussion by DJBrause** [Khopa/dcs_liberation#2344] [Q&A] [answered]
Hello,

I'd like to help and contribute to this project.

I have pretty good Python knowledge, but I'm happy to help in any way possible.

Can anyone please let me know what is needed?

Thank you.

**Comment by DanAlbert** [comment, marked as answer]
Best way is to ping me (ColonelPanic) in our Discord. I'll get you added to the dev channel and we can chat more easily.

The best way to contribute IMO is to pick the things that interest you and work on those. Pick something small that drives you nuts when playing and fix that. I do recommend picking something small to start with :) There's a lot of code to learn and quality definitely varies (the newer the better, usually). If you have vague ideas but don't quite know where to start, I or one of the other devs should be able to give you pointers.

A word of warning, a bunch of the core dev team (the people that can review and merge your PRs) is very busy IRL right now, so PR latency varies a lot. Small things like easy bug fixes are still quick, but something that takes real brain power to review can take a while.
